src/app/static/src/tests/integration/surveyAndProgram.itest.ts

```diff
@@ -0,0 +1,32 @@
+import {actions} from "../../app/store/surveyAndProgram/actions";
+
+const fs = require("fs");
+const FormData = require("form-data");
+
+describe("Survey and program actions", () => {
+
+    beforeEach(() => {
+        fs.writeFileSync("fakesurvey");
+    });
+
+    afterEach(() => {
+        fs.unlinkSync("fakesurvey")
+    });
+
+    it("can upload survey", async () => {
+
+        const commit = jest.fn();
+
+        const file = fs.createReadStream("fakesurvey");
+        const formData = new FormData();
+        formData.append('file', file);
+
+        await actions.uploadSurvey({commit} as any, formData);
+
+        expect(commit.mock.calls[0][0]).toStrictEqual({
+            type: "SurveyError",
+            payload: "could not find function \"validate_func\""
```

Comment:
Is this the current error coming out of hintr? we expect this to turn into something more meaningful when hintr is further along?

Reply:
This is the current error from hintr :/